### PR TITLE
Rewriting Data Cleaning Vignette

### PR DESCRIPTION
**Pull request by ruckeran**
content:
1. what is the goal there? 
2. how is it achieved? 

writing:
more in a passive form
focus on the R functions

+ use case for checkMissingsByValues()

### CODE REVIEW

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 1]
```diff
@@ -80,61 +80,62 @@ The values `-97`, `-98` and `-99` are now tagged as `miss`.
 
 
 ## Recode IDs
-To ensure that the IDs remain the same across multiple subsets and describe the same person, we recode them using a template. The first step is to create the path for the template.
+To ensure that the IDs remain the same across multiple subsets and describe the same person, they are recoded using a template. First, a path for the template is created.
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:84]: Could a short explanation be added, why identifier variables are recoded in the first place?

File: vignettes/FDZ_Data_Cleaning.Rmd [hunk 2]
```diff
@@ -166,8 +166,8 @@ extractMeta(dat7, vars = c("books"))
 
 
 ## Anonymization: Form grouping variable
-For anonymization reasons, we partially form grouping variables for items with a case number <6. To do this, we create a new _FDZ variable by "cloning" the variable we want to group with `cloneVariable()`. For `varName` we enter the original variable name, for `new_varName` we enter the variable name including the appendix "_FDZ". The variable label is automatically adjusted with the option `label_suffix`.
-With `recodeGADS()` the old values (`oldValues`) are replaced by new ones (`newValues`). With the help of `changeValLabels()` the value labels of the new groupings are changed.
+Small case numbers pose a risk of anonymity. For variables with a number of cases <6, which are not ratings, feelings or opinions, it is advisable to combine small numbers of cases into meaningful groups. To do so, `cloneVariable()` creates a new _FDZ variable by cloning the variable to be grouped. For `varName` the original variable name is entered, for `new_varName` the variable name including the appendix "_FDZ". The variable label is automatically adjusted with the option `label_suffix`.
```
Comment (beckerbenj) [inline vignettes/FDZ_Data_Cleaning.Rmd:170]: I like the wording a lot! Could it emphasized, however, that such pseudonymization is only applied to variables that contain sensitive information about individuals? (German term: Personenbezug)

**Review by beckerbenj — APPROVED** [review]
Really like the new wordings and style 👍 Two minor nits in the comments below. 

Feel free to address them. Please let me know if you want to merge the PR; I think there may be some merge conflicts and/or issues with the CI workflow.

File: vignettes/FDZ_Data_Cleaning.Rmd (hunk 1 above)
Comment (ruckeran) [reply vignettes/FDZ_Data_Cleaning.Rmd:84]: sounds good!

File: vignettes/FDZ_Data_Cleaning.Rmd (hunk 2 above)
Comment (ruckeran) [reply vignettes/FDZ_Data_Cleaning.Rmd:170]: thanks! will do.

**Review by beckerbenj — APPROVED** [review]

**Pull request merged**